pkgs/applications/networking/p2p/tremc/default.nix

```diff
@@ -1,13 +1,11 @@
 { lib, stdenv, fetchFromGitHub, python3Packages
 , x11Support ? !stdenv.isDarwin
-, xclip ? null
-, pbcopy ? null
+, xclip
 , useGeoIP ? false # Require /var/lib/geoip-databases/GeoIP.dat
 }:
 let
   wrapperPath = with lib; makeBinPath (
-    optional x11Support xclip ++
-    optional stdenv.isDarwin pbcopy
+    optional x11Support xclip
```

Comment:
This change looks wrong.

Reply:
Hmm, so it could be that someone might have overridden this package to pass a valid `pbcopy` derivation. Because there's no `pbcopy` in nixpkgs itself. I do know that macOS has a `pbcopy` binary in its PATH by default though.

Reply:
Then this is correct. If someone has overwriten this, the person needs to update their overlay.